docs/table_design/table_types/primary_key_table.md

```diff
@@ -129,7 +129,7 @@ PROPERTIES("replication_num" = "3",
   > - If you want to modify this parameter after the table is created, please see the part Modify the properties of table in [ALTER TABLE](../../sql-reference/sql-statements/data-definition/ALTER_TABLE.md).
   > - It is recommended to set this property to true if the disk is SSD.
   > - As of version 2.3.0, StarRocks supports to set this property.
-  > - StarRocks's shared-data mode does not support enabling the persistent index for a Primary Key table.
+  > - Since v3.1.4, StarRocks's shared-data mode supports enabling the persistent index for a Primary Key table.
```

Comment:
supports enabling the persistent index **on local disks** for a Primary Key table.